benches/benches/bevy_reflect/path.rs

```diff
@@ -0,0 +1,94 @@
+use std::{fmt::Write, str, time::Duration};
+
+use bevy_reflect::ParsedPath;
+use criterion::{
+    black_box, criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion, Throughput,
+};
+use rand::{distributions::Uniform, Rng, SeedableRng};
+use rand_chacha::ChaCha8Rng;
+
+criterion_group!(benches, parse_reflect_path);
+criterion_main!(benches);
+
+const WARM_UP_TIME: Duration = Duration::from_millis(500);
+const MEASUREMENT_TIME: Duration = Duration::from_secs(2);
+const SAMPLE_SIZE: usize = 500;
+const NOISE_THRESHOLD: f64 = 0.03;
+const SIZES: [usize; 6] = [100, 3160, 1000, 3_162, 10_000, 24_000];
+
+fn deterministic_rand() -> ChaCha8Rng {
+    ChaCha8Rng::seed_from_u64(42)
```

Comment:
nitpick: would also accept inlining this since it's only used in one place, optionally moving the seed to a named const, but not necessary.

Reply:
I re-used the function from `benches/benches/bevy_ecs/world/world_get.rs`

https://github.com/bevyengine/bevy/blob/b88ff154f24314539fc7f03572215bd0283d98ee/benches/benches/bevy_ecs/world/world_get.rs#L28-L30